app/src/main/res/layout/dialog_date_range_transaction.xml

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
+        android:id="@+id/scrim"
+        android:background="@color/transparent_white"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        >
+
+        <com.google.android.material.navigation.NavigationView
```

Comment:
@danacealfonso 
`NavigationView` is not intended for this. The `ConstraintLayout` is enough.